### DISCUSSION TITLE
One inquiry about GPU

### DISCUSSION THREAD

**Discussion by yoonk0511** [torchmd/torchmd#23] [Q&A]
Hello, 
I was searching for appropriate simulation kits in my research, and Fortunately I found the torchmd.
Actuatlly, I installed it and run some tutorials. But I have a question about GPU usage.
I wonder that only TorchMD-NET uses GPU, or TorchMD(Simulation) can also simulate with GPU .

**Comment by stefdoerr** [comment]
Yes. both should support GPU. But don't expect great simulation speed from TorchMD. It's orders of magnitude slower than simulating with a real dedicated MD code
![image](https://user-images.githubusercontent.com/7935362/184825436-aa753316-43cb-4dac-a1c0-78ad6564a1e5.png)
